bench_scripts/ppl.py

```
@@ -0,0 +1,34 @@
+    preset=sys.argv[1]
+    import pandas as pd
+    ds=pd.read_parquet("/home/larkinwc/wikitext2_test.parquet")
+    text="\n\n".join(t for t in ds["text"].tolist() if t.strip())
+    kw=dict(model="Qwen/Qwen3.5-9B", tensor_parallel_size=4, dtype="float16",
+            enforce_eager=True, gpu_memory_utilization=0.70, max_model_len=4096,
+            language_model_only=True)
+    if preset!="auto": kw["kv_cache_dtype"]=preset
+    llm=LLM(**kw)
+    tok=llm.get_tokenizer()
+    ids=tok(text, add_special_tokens=False)["input_ids"]
+    W=512; stride=512; N=24  # 20 windows of 2048 = ~40k tokens evaluated
```

Ruff (0.15.20)

[error] Multiple statements on one line (semicolon)

(E702)